### ISSUE TITLE
Feature: Allow set default job image via env var

### ISSUE THREAD

**Issue by piotrminkina** [k3s-io/helm-controller#213]
Hello,

I would like to use Helm Controller in an enterprise environment without open internet access. All Docker images used on the Kubernetes cluster come from our internal image registry and there is no way to deploy images from other sources. Currently, you allow the job image to be set using the `spec.jobImage` field in the HelmController resource, but in my case this forces us to overwrite it in every deployed resource of this type.

https://github.com/k3s-io/helm-controller/blob/32c2e08fa0265cbc36680d50ca766c5a73cecd85/pkg/controllers/chart/chart.go#L56

Please allow me to change the default image for Job by setting an environment variable on the HelmController container. Thank you in advance for this possibility.

Regards
Piotr Minkina

**Comment by brandond** [comment]
PR welcome!

**Comment by piotrminkina** [comment]
@brandond done :)

**Comment by piotrminkina** [comment]
@brandond The Docker image for version v0.15.5 has not yet been built. Shouldn't this happen automatically when the new version is released?

**Comment by brandond** [comment]
Yes, but there was some maintenance to our CI infra that appears to have removed the GH API token. I've opened an issue and will re-run CI when it is resolved.
